### PR TITLE
Doc: PCRE->PCRE2, libgit info, and Power

### PR DESCRIPTION
**Pull request by PallHaraldsson**
[skip ci]

Hi @vtjnash I guess you did "vtjnash merged commit 70fff5b into JuliaLang:master 16 minutes ago"

[I see now you really did this yesterday, not a few minutes ago..]

That is not a problem.. I however did:

git push -f

on my branch patch-12 after:

error: failed to push some refs to 'https://github.com/PallHaraldsson/julia.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

And as you can see: 

"vtjnash merged 0 commits into JuliaLang:master from PallHaraldsson:patch-12"

That doesn't make sense.. I thought I could only mess up my own, PR.. not "merge" or close it..

I just want to know I didn't mess up master and/or GitHub..

Above contradicts:

https://github.com/JuliaLang/julia/pulls?utf8=%E2%9C%93&q=is%3Apr%20PCRE

I was trying to get out of problems similar to here (I was up to 11 commits here):

https://github.com/JuliaLang/julia/pull/17430

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -280,10 +280,10 @@ Julia uses the following external libraries, which are automatically downloaded
 - **[SuiteSparse]** (>= 4.1) — library of linear algebra routines for sparse matrices.
 - **[ARPACK]**               — collection of subroutines designed to solve large, sparse eigenvalue problems.
 - **[FFTW]** (>= 3.3)        — library for computing fast Fourier transforms very quickly and efficiently.
-- **[PCRE]** (>= 10.00)      — Perl-compatible regular expressions library.
+- **[PCRE2]** (>= 10.00)     — Perl-compatible regular expressions library.
 - **[GMP]** (>= 5.0)         — GNU multiple precision arithmetic library, needed for `BigInt` support.
 - **[MPFR]** (>= 3.0)        — GNU multiple precision floating point library, needed for arbitrary precision floating point (`BigFloat`) support.
-- **[libgit2]** (>= 0.23)    — Git linkable library, used by Julia's package manager
+- **[libgit2]** (>= 0.23)    — a permissively linkable Git library, used by Julia's package manager
```
Comment (kshyatt) [inline README.md]: Would it be worth actually linking to `libgit2`'s license right here?

Comment (StefanKarpinski) [inline README.md]: https://github.com/libgit2/libgit2/blob/master/COPYING – i.e. GPL with linking exception. Not sure what this vague description is supposed to mean.

Comment (StefanKarpinski) [inline README.md]: Here we generally just describe the library, not it's license. Links to licenses go in the LICENSE.md file.

Comment (kshyatt) [inline README.md]: I'm just saying if we're going to talk about the license anyway, "permissive" is a word that will get you 10 different answers if you ask 8 people. Wherever we talk about `libgit2`'s license, we should link it, I think.

Comment (StefanKarpinski) [inline README.md]: Agree – I think we should take out the mention of the license here altogether.

Comment (StefanKarpinski) [inline README.md]: And "permissive" is typically used to describe MIT/BSD-style licenses, not LGPL or GPL with linking exception.

Comment (PallHaraldsson) [inline README.md]: How about just "Git library"? Skip "linkable", that I felt "explaining", but neither needs be here?

**Pull request merged**